### ISSUE TITLE
forward-over-reverse gives wrong gradgrad for `Softsign`

### ISSUE THREAD

**Issue by upupyy** [pytorch/pytorch#80926]
### 🐛 Describe the bug

forward-over-reverse gives wrong gradgrad for `Softsign`

```py
import torch

def get_fn():
    arg_class = torch.nn.Softsign()
    def fn(input):
        fn_res = arg_class(input)
        return fn_res
    return fn
fn = get_fn()
input_tensor = torch.tensor([-0.3606-0.7886j, -0.4246-0.1352j], dtype=torch.complex128)

try:
    input = input_tensor.clone().detach().to('cpu').requires_grad_()
    torch.autograd.gradgradcheck(fn, (input), atol=0.01, rtol=0.01, check_fwd_over_rev=True, check_rev_over_rev=False, check_undefined_grad=False)
except Exception as e:
    print('fwdrev fail')
    print(e)

try:
    input = input_tensor.clone().detach().to('cpu').requires_grad_()
    torch.autograd.gradgradcheck(fn, (input), atol=0.01, rtol=0.01, check_fwd_over_rev=False, check_rev_over_rev=True, check_undefined_grad=False)
    print('revrev success')
except Exception as e:
    print(e)
```

```
fwdrev success
While considering the imaginary part of complex inputs only, Jacobian computed with forward mode mismatch for output 0 with respect to input 0,
numerical:tensor([[-0.0377+0.4046j,  0.0000+0.0000j],
        [ 0.0000+0.0000j, -0.1065+0.1221j]], dtype=torch.complex128)
analytical:tensor([[0.2128+0.4046j, 0.0000+0.0000j],
        [0.0000+0.0000j, 0.0129+0.1221j]], dtype=torch.complex128,
       grad_fn=<CopySlices>)

revrev success
```

Obviously, `Softsign` has well-defined second-order gradient. Thus, I think fwd-over-rev has some issues since rev-over-rev can pass the `gradgradcheck`

### Versions

pytorch: 1.11.0

cc @ezyang @albanD @zou3519 @gqchen @pearu @nikitaved @soulitzer @Lezcano @Varal7

**Comment by upupyy** [comment]
Yes, it is fixed in the nightly version. Let me close it
